meta-schemas/clocks.yaml

```diff
@@ -8,7 +8,9 @@ $schema: "http://json-schema.org/draft-07/schema#"
 
 properties:
   clocks:
-    $ref: "cell.yaml#array"
+    anyOf:
+      - $ref: "cell.yaml#array"
+      - type: object
```

Comment:
This is saying the schema 'clocks' is an object which is always the case unless you have 'clocks: true', not that it's a node.

It should be '$ref: nodes.yaml#'

Also, I think you can use oneOf rather than anyOf.